README.md

```diff
@@ -118,6 +118,27 @@ Then to verify the pipeline definition using the defined policies:
 
     make check-pipeline
 
+### Running policies against local [ec-cli] build
+
+Build a local version of `ec-cli` in your local ec-cli repo:
+
+    make build
+
+Create a `policy.yaml` file in your local `ec-cli` repo with something like:
+
+    ---
+    sources:
+      - policy:
+        - github.com/enterprise-contract/ec-policies//policy//lib
+        - github.com/enterprise-contract/ec-policies//policy//release
+      data:
+        - oci::quay.io/konflux-ci/tekton-catalog/data-acceptable-bundles:latest
+        - github.com/release-engineering/rhtap-ec-policy//data
+
+Run the locally built `ec-cli` command
+
+    dist/ec_<arch> validate image --verbose --images '{"components": [{"containerImage": "<container-image>", "name":"my-image"}]}' --policy 'policy.yaml' --public-key <public-key-to-verify-the-image> --strict false  --ignore-rekor --verbose | jq
```

Comment:
We might want the source reference here, e.g.:
```suggestion
    dist/ec_<arch> validate image --verbose --images '{"components": [{"containerImage": "<container-image>", "name":"my-image", "source":{"git":{"url":"<repository-url>","revision":"<commit-id>"}}}]}' --policy 'policy.yaml' --public-key <public-key-to-verify-the-image> --strict false  --ignore-rekor --verbose | jq
```

Perhaps use the text output?

```suggestion
    dist/ec_<arch> validate image --verbose --images '{"components": [{"containerImage": "<container-image>", "name":"my-image"}]}' --policy 'policy.yaml' --public-key <public-key-to-verify-the-image> --strict false  --ignore-rekor --verbose --output=text
```

Reply:
Done